pset-iterators.js

```diff
@@ -138,6 +172,13 @@ console.log('Problem 5:')
 
 // Add your code below this line
 
+const livingPresidents = presidents.filter((president) => {
+	return president.death_year === null
+}).map((president) => {
+    return president.president
+})
+
+console.log(`Currently, the living presidents are: ${livingPresidents}.`)
```

Comment:
@cswebdesign Good job here using the filter method to return the president's who are still alive. In this use case, you didn't need a map since you are returning an array of objects the filter method would've been enough. Now if they asked to print out the name then chaining would've been great for that use case, But for this one filter was enough

Take a look at this approach [here]( https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L148)